paddle/fluid/inference/api/paddle_pass_builder.h

```diff
@@ -117,6 +117,8 @@ class CpuPassStrategy : public PassStrategy {
 
       for (auto &pass : std::vector<std::string>(
                {"depthwise_conv_mkldnn_pass",    //
+                "conv_bn_fuse_pass",             //
```

Comment:
please add some explanation here, why need `conv_bn_fuse_pass` again in mkldnn_passes.